src/Enigma2.cpp

```diff
@@ -299,11 +299,10 @@ bool Enigma2::OpenLiveStream(const PVR_CHANNEL &channelinfo)
       // Zapping is set to true, so send the zapping command to the PVR box
       std::string strServiceReference = m_channels.GetChannel(channelinfo.iUniqueId)->GetServiceReference().c_str();
 
-      std::string strTmp;
-      strTmp = StringUtils::Format("web/zap?sRef=%s", WebUtils::URLEncodeInline(strServiceReference).c_str());
+      std::string strCmd = StringUtils::Format("web/zap?sRef=%s", WebUtils::URLEncodeInline(strServiceReference).c_str());
```

Comment:
`const std::string`?

Reply:
I'll go through the go base and clean this up in general. Been meaning to do it for a while.